src/main/java/com/flightstats/hub/metrics/StatsDFilter.java

```diff
@@ -0,0 +1,55 @@
+package com.flightstats.hub.metrics;
+
```

Comment:
Another note that the actual filtering logic will change, but is in the scope of another PR.